uint64 timestamp 		# time since system start (microseconds)
uint8 is_offboard # mode
uint64 cmd1
uint64 cmd2
uint64 cmd3
